packages/storybook/stories/Setup.stories.mdx

```diff
@@ -0,0 +1,129 @@
+import { Meta, Source } from '@storybook/addon-docs';
+
+import indexText from '!!raw-loader!./snippets/ReactIndex.snippet.tsx';
+
+<Meta
+  id="setup-communication-react"
+  title="Setup"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+# Setting up your app to use Azure Communications UI Library
+
+Currently the Azure Communications UI library supports react versions `>= 16.8.0` or ` < 18`
+
+Using the UI library in react you will need to downgrade the version of react that you are
```

Comment:
When following this setup guide, the react version bundled with the `create-react-app` will need to be downgraded from react 18 to a supported react version between `>= 16.8.0` or ` < 18`. This page will cover which packages and files to downgrade and edit to get you up and running.